src/align.cpp

```diff
@@ -518,7 +517,7 @@ void align_right_comments(void)
                        pc->orig_col, prev->orig_col_end, cpd.settings[UO_align_right_cmt_gap].n);
                skip = true;
             }
-            if (!skip)
+            if (skip == false)
```

Comment:
Why is it better? I know many people use it but in my opinion it is bad style. It requires two thoughts my version just one. And the compiler doesn't care.